planning/autoware_obstacle_stop_planner/schema/obstacle_stop_planner.schema.json

```
@@ -0,0 +1,322 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameter for obstacle stop planner",
+  "type": "object",
+  "definitions": {
+    "obstacle_stop_planner": {
+      "type": "object",
+      "properties": {
+        "chattering_threshold": {
+          "type": "number",
+          "description": "INSERT_DESCRIeven if the obstacle disappears, the stop judgment continues for chattering_threshold [s]PTION",
+          "default": "0.5"
+        },
+        "max_velocity": {
+          "type": "number",
+          "description": "max velocity [m/s]",
+          "default": "20.0"
+        },
+        "ego_nearest_dist_threshold": {
+          "type": "number",
+          "description": "[m]",
+          "default": "3.0"
+        },
+        "ego_nearest_yaw_threshold": {
+          "type": "number",
+          "description": "[rad] = 60 [deg]",
+          "default": "1.046"
+        },
+        "enable_slow_down": {
+          "type": "boolean",
+          "description": "whether to use slow down planner [-]",
+          "default": "False"
+        },
+        "enable_z_axis_obstacle_filtering": {
+          "type": "boolean",
+          "description": "filter obstacles in z axis (height) [-]",
+          "default": "True"
+        },
+        "z_axis_filtering_buffer": {
+          "type": "number",
+          "description": "additional buffer for z axis filtering [m]",
+          "default": "0.0"
+        },
+        "voxel_grid_x": {
+          "type": "number",
+          "description": "voxel grid x parameter for filtering pointcloud [m]",
+          "default": "0.05"
+        },
+        "voxel_grid_y": {
+          "type": "number",
+          "description": "voxel grid y parameter for filtering pointcloud [m]",
+          "default": "0.05"
+        },
+        "voxel_grid_z": {
+          "type": "number",
+          "description": "voxel grid z parameter for filtering pointcloud [m]",
+          "default": "100000.0"
+        },
+        "use_predicted_objects": {
+          "type": "boolean",
+          "description": "whether to use predicted objects [-]",
+          "default": "False"
```

Could you make it to lower case?
It seems it causes some error when converting to markdown if false/true are capitalized.

```suggestion
          "default": "false"
```